src/node/element.rs

```diff
@@ -86,7 +92,15 @@ macro_rules! impl_elements {
                     fn child_nodes(&self) -> Arc<NodeList> {
                         self.node_behavior.child_nodes()
                     }
-                }
+
+                    fn tag_name(&self) -> String {
+                        String::from($tag)
```

Comment:
Nit:

```suggestion
                        $tag.to_string()
```

For code style 💇